src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+
+        Colors[] colorValues = Colors.values();
+        Random rand = new Random();
+        int randomIndex = rand.nextInt(colorValues.length - 1);
+
+        return String.valueOf(colorValues[randomIndex]);
```

Comment:
Instead of using `String.valueOf`, it's recommended to use the `name()` method to get the string representation of enum constants. This ensures that even if the `toString()` method is overridden, the actual name of the enum constant is returned.